tdl-ksrd/app/models/user.rb

```diff
@@ -1,18 +1,27 @@
 require 'bcrypt'
 
-class User < ActiveRecord::Base
+class User < ApplicationRecord
 
   include BCrypt
 
   has_secure_password
 
   before_save { self.email = email.downcase }
 
+  before_create :add_hash_for_url
+
+  has_many :comments
+
   validates :name, :presence => true,
             :length => { :maximum => 40 }
 
   validates :email, :presence => true, uniqueness: { case_sensitive: false }
 
   validates :password, :length => { :within => 6..50 }
 
+  def add_hash_for_url
+    self.hash_for_url = Sysrandom.hex(32)
+  end
```

Comment:
esto lo probaron? recien quise registrarme y me tiro: NoMethodError (undefined method `hash_for_url=' for #<User:0x00007fed7073ad38>)

Reply:
me faltaba el db:setup, mala mia! Gracias @ChrisFlorez